### DISCUSSION TITLE
Need to know the stable fluentd-kubernetes-daemonset version compatible with elasticsearch 7.17.6, as current version 1.12.3 is having bugs

### DISCUSSION THREAD

**Discussion by YPuri** [community/community#85320] [Programming Help]
### Select Topic Area

Question

### Body

In my project we are using Fluentd-kubernetes-daemonset version 1.12.3 for openshift4.12 nodes for log colllection and fluentd_es version latest for RHEL machines with Elasticsearch 7.17.6. I am facing error in fluentd_es "failed to flush buffer" and "noaddress for elasticsearch-sample-es-data-0.elasticsearch-sample-es-datq.elastic.svc". Initially fluentd is able to send logs to elasticsearch for 2-3 days and after that this problem is occurring. So, i dont think its a dns issue.For resolving this issue I referred old github discussions and there version change was suggested but there is no clarity about the recommended version. So, I need to know a stable version for fluentd compatible with elasticsearch7.17.6
Kindly share your recommendations.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @YPuri ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the GitHub product Discussions. 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
